templates/base.html

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        {% block css %}
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootswatch/4.5.2/sketchy/bootstrap.min.css">
+        {% endblock %}
```

Comment:
Why does this need to be in block?

Reply:
The formatting of this currently is incorrect for other templates inheriting the css. These stylesheets will be moved outside of the block, but the block will stay so when I add css to other templates inheriting from base.html, that css will be within the html head tag.